packages/fxa-content-server/server/lib/beta-settings.js

```diff
@@ -0,0 +1,113 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const { readFileSync } = require('fs');
+const { join } = require('path');
+const { createProxyMiddleware } = require('http-proxy-middleware');
+const config = require('./configuration');
+
+let settingsIndexFile;
+const env = config.get('env');
+const settingsIndexPath = join(
+  __dirname,
+  '..',
+  '..',
+  config.get('static_directory'),
+  'beta',
+  'settings',
+  'index.html'
+);
+
+if (env !== 'development') {
+  settingsIndexFile = readFileSync(settingsIndexPath, {
+    encoding: 'UTF-8',
+  });
+}
+
+const settingsConfig = {
+  env,
+  sentry: {
+    dsn: config.get('sentry.client_errors_dsn'),
+  },
+  servers: {
+    gql: {
+      url: config.get('settings_gql_url'),
+    },
+    auth: {
+      url: config.get('fxaccount_url'),
+    },
```

Comment:
These two server configs are not in `fxa-content-server/server/lib/configuration.js` AFAICT.

Reply:
hahaha never mind 🤦